code/modules/mob/new_player/sprite_accessories.dm

```diff
@@ -1,4 +1,4 @@
-/*
+/paradise_schema.sql/*

```

Comment:
...i...don't event know why that is like that......unless i accidently delete an *